fusor-ember-cli/app/templates/components/auto-node-registration-mac-address.hbs

```diff
@@ -7,8 +7,13 @@
       {{hostNumber}}.
     </label>
     {{#if multipleMacAddresses}}
-      {{view "select" id=selectId class="form-control new-node-detect-select"
-        prompt="Select an interface" disabled=deselected content=host.macAddresses value=host.value}}
+      {{#x-select value=host.value id=selectId class="form-control new-node-detect-select"
+        prompt="Select an interface" disabled=deselected}}
+        {{#each host.macAddresses as |opt| }}
+          {{#x-option value=opt}}{{opt}}{{/x-option}}
```

Comment:
I couldn't get `auto-node-registration-mac-address` to show in my dev setup for multiple mac addresses, so this refactoring is just a guess.